### ISSUE TITLE
GitHub repositories processing issue

### ISSUE THREAD

**Issue by Tech-Overlord** [sourcerer-io/sourcerer-app#370]
Hi,

I have about 7 repositories. Each time that I perform an "Update GitHub Repos", it says that all 7 of the repositories have been processed successfully. However, their are only 4 git repos visible on my sourcerer profile. Also the analysis on my Sourcerer profile is based off of those 4 repositories only. I am wondering why all the other repository stats, commits etc are not being fetched?

Regards,
Ali

**Comment by reisraff** [comment]
Also having the same issue, It can not process my C repo, reisraff/cim, that has some commits, and it does not count since it is not processed.

**Comment by tuliobluz** [comment]
I'm facing the same problem, I have five repo, but It is showing just one. I tried to create a new account, but It has happened the same problem.

**Comment by gaui** [comment]
Same here with GitHub. 

First run: 8 repos ok, 24 skipped.
After couple of runs: 1 repo ok, 31 skipped.

Maybe a GitHub API limitation?

**Comment by VentGrey** [comment]
Again, same issue here.

**Comment by vinzlercodes** [comment]
Yes same even I have 2 public repo but my jupyter lab repository does not seem to be getting processed

**Comment by ahmad-ali14** [comment]
same here: https://sourcerer.io/ahmad-ali14
